AVAL-Spec.yml

```diff
@@ -491,9 +491,9 @@ definitions:
       requiredDocuments:
         type: "array"
         description: "document types as Defined in AvalDocuments/fileType"
+        example: "[AD.1.1.101]"
```

Comment:
The quotation marks are not set correctly for an array of strings. The expamle looks like a single string value. 

I suggest to use the following example `[ "AD.1.1.101", "AD.1.9.901"]`.

Reply:
yes; I fixed that